src/components/Popups/ReplyField/MentionItem.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { bdlBoxBlue } from 'box-ui-elements/es/styles/variables';
+import { ContentState } from 'draft-js';
+
+type Props = {
+    children: React.ReactNode;
+    contentState: ContentState;
+    decoratedText: string;
+    entityKey?: string;
+};
+
+const MentionItem = ({ contentState, entityKey, children }: Props): JSX.Element => {
+    const id = entityKey ? contentState.getEntity(entityKey).getData().id : '';
+
+    return (
+        <a href={`/profile/${id}`} style={{ color: bdlBoxBlue }}>
```

Comment:
Can this use SCSS or does it need to be an inline style? Does this path work for all usecases?

Reply:
> Can this use SCSS or does it need to be an inline style? Does this path work for all usecases?

Path? If you mean the href link, yeah, both `UserMini` and `GroupMini` have required `id`.